### PR TITLE
add rocksdb metrics

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Add metrics to monitor the size of each rocksdb's column families. We will add more metrics about rocksdb's internal statistics after rocksdb supply more C api.

@siddontang @BusyJay @disksing PTAL

### CODE REVIEW

File: src/raftstore/store/metrics.rs [hunk 1]
```diff
@@ -75,4 +75,11 @@ lazy_static! {
             "Total nanoseconds spent in raft processing.",
             &["type"]
         ).unwrap();
+
+    pub static ref ENGINE_STATISTICS_GAUGE_VEC: GaugeVec =
```
Comment (siddontang) [inline src/raftstore/store/metrics.rs]: This is the statistics for RocksDB, not for raftstore. I think we should move this to another place, like util/rocksdb

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -1453,6 +1454,22 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.register_compact_lock_cf_tick(event_loop);
     }
 
+    fn on_dump_rocksdb_metrics(&mut self, event_loop: &mut EventLoop<Self>) {
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: same reason, it is not suitable to get rocksdb statistics in raftstore mod. 
You can use a function to get all rocksdb statistics, then use this function here.

**Comment by siddontang** [comment]
LGTM

File: src/raftstore/store/store.rs [hunk 3]
```diff
@@ -1337,6 +1337,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 .get_property_int_cf(handle, ROCKSDB_TOTAL_SST_FILE_SIZE_PROPERTY)
                 .expect("rocksdb is too old, missing total-sst-files-size property");
 
+            // It is important to monitor each cf's size, especially the "raft" and "lock" column
+            // families.
+            STORE_SIZE_GAUGE_VEC.with_label_values(&[cf]).set(cf_used_size as f64);
```
Comment (disksing) [inline src/raftstore/store/store.rs]: You should define another metrics. Cf name is different from the old types `[capacity, available]`.

**Comment by disksing** [comment]
LGTM

**Pull request merged**